README.md

```diff
@@ -1,96 +1,127 @@
+- In the terminal, type `blender` and press enter.  This will launch blender from the terminal.
 
-## Installation
-- Download the zipped data file containing SMPL Model from the link provided to you at the time you recevied a link to this repo. If you do not have this link, please contact support@meshcapade.com for help.
-- Place the data file (it will be a file with naming of the form: `smplx_blender_YYYYMMDD.zip`) inside the folder containing the SMPL blenderadd-on code. Do not unzip.
-- Within Blender (2.92+), under Edit -> Preferences navigate to the Add-ons tab
-- Click the Install button on the upper right, navigate to and select the downloaded zip file, and click 'Install Add-on'
-- Once loaded, check the box next to the now appeared add-on title to activate.
 
-![image](https://user-images.githubusercontent.com/538382/131877148-3d65f453-13ef-4c47-b56f-fd008930937a.png)
+## Getting avatars inside Blender
 
-Enable sidebar in 3D Viewport>View>Sidebar, the tabs SMPL-X and Meshcapade Utilities should show along the sidebar.
+There are a few ways to get Meshcapade avatars into Blender.  One way is to create and download free avatars from our platform, meshcapade.me.  You can create up to 5 free avatars a day, or you can purchase credits to be able to do more than that.  
 
-![image](https://user-images.githubusercontent.com/538382/131878699-df5b7fd1-9bbc-47ae-9cb4-8fd319727c9d.png)
+Academic or commercial license customers can get an additional component to the plugin that allows them to add an unlimited number of SMPL bodies directly to their scenes.  They can either create avatars from scratch, or they can load .npz files that contain not only body shape definitions but animations as well.  Both of these methods support SMPLH (to a limited extent), SMPLX and SUPR bodies and you can create an unlimited number of avatars using these two methods.
 
+## Pose Correctives
 
-## Make Addon Release
+Meshcapade avatars have a built in component that allows for statistically accurate pose based deformations.  Once a Meshcapade avatar is in your Blender scene, you can animate or pose it as you normally would.  Click the `Calculate Pose Correctives` button for a single frame or `Calculate Pose Correctives for Entire Sequence` for an animation sequence.  This feature is available for non licensed use of our Blender plugin.
 
-Run:
+Note: A bug exists in Blender up to and including LTS 6.1 in which imported .fbx files have their blend shapes clamped to 0 and 1.  The pose correctives can have values that are less than 0 and greater than 1, so this is a problem here.  After importing your .fbx, click the `Fix Blend Shape Ranges` button on the plugin and it will set all the blend shape ranges to the highest and lowest possible values blender allows: 10 and -10.  This is a known issue which Blender has already addressed, it just hasn’t been released as of my writing this.
```

Comment:
I was talking about LTS 6.1 - https://www.blender.org/download/lts/3-6/ - then scroll down.  Of course they've released LTS 6.2 since then.  The point is that was the latest version that was out at the time of writing and it still wasn't fixed.  They are always releasing stuff so I don't want to check again.

Reply:
I see, I think for me the confusion was in the naming, I wasn't sure what 6.1 refers to but now I see that it refers to 3.6.1.

Reply:
OH. My goodness, sorry.  Yes you're absolutely right.